_includes/templates/forms/plain-writing-feedback-2-form.html

```diff
@@ -0,0 +1,294 @@
+                           required>
+                </div>
+            </div>
+
+            <div class="form-group">
+                <div class="form-l_col-1-2">
+                    <label class="form-label-header" for="plain-writing-feedback_last-name">
+                        Last name
+                    </label>
+                </div>
+                <div class="form-l_col-1-2">
+                    <input class="input__long"
+                           id="plain-writing-feedback_last-name"
+                           type="text"
+                           placeholder="Requestor's last name"
+                           required>
+                </div>
+            </div>
+
+            <div class="form-group">
+                <div class="form-l_col-1-2">
+                    <label class="form-label-header" for="plain-writing-feedback_address-1">
+                        Address 1
+                    </label>
+                </div>
+                <div class="form-l_col-1-2">
+                    <input class="input__long"
+                           id="plain-writing-feedback_address-1"
+                           type="text"
+                           placeholder="Requestor's address 1"
+                           required>
+                </div>
+            </div>
+
+            <div class="form-group">
+                <div class="form-l_col-1-2">
+                    <label class="form-label-header" for="plain-writing-feedback_address-2">
+                        Address 2
+                    </label>
+                </div>
+                <div class="form-l_col-1-2">
+                    <input class="input__long"
+                           id="plain-writing-feedback_address-2"
+                           type="text"
+                           placeholder="Requestor's address 2"
+                           required>
+                </div>
+            </div>
+
+            <div class="form-group">
+                <div class="form-l_col-1-2">
+                    <label class="form-label-header" for="plain-writing-feedback_city">
+                        City
+                    </label>
+                </div>
+                <div class="form-l_col-1-2">
+                    <input class="input__long"
+                           id="plain-writing-feedback_city"
+                           type="text"
+                           placeholder="Requestor's city"
+                           required>
+                </div>
+            </div>
+
+            <div class="form-group">
+                <div class="form-l_col-1-2">
+                    <label class="form-label-header" for="plain-writing-feedback_state">
+                        State
+                    </label>
+                </div>
+                <div class="form-l_col-1-2">
+                    <div class="select-content">
+                        <select id="plain-writing-feedback_state">
```

Comment:
We might want to save an array of states to make this a template rather than add every single state on every single form. Ex https://gist.github.com/jimmynotjim/8446233